lib/ansible/modules/files/copy.py

```diff
@@ -46,6 +46,7 @@
     - If C(dest) is a non-existent path and if either C(dest) ends with "/" or C(src) is a directory, C(dest) is created.
     - If I(dest) is a relative path, the starting directory is determined by the remote host.
     - If C(src) and C(dest) are files, the parent directory of C(dest) is not created and the task fails if it does not already exist.
+    - If C(content) is used and C(dest) is a file, file is created if it does not already exist.
```

Comment:
In my opinion, this is documenting obvious functionality of the module. As such, I don't find this added documentation useful.

Perhaps additional clarification can be added to the description of the module to indicate that it does not require a file.

Reply:
I made a change to the content description now, @sivel please review

Reply:
@acozine Check now